app/assets/stylesheets/sephora_style_guide/variables/_icons.scss

```diff
@@ -45,7 +45,7 @@ $icon_100x30-classes: cimbclick permata hongleong maybank;
 $icon_90x40-width: 90px;
 $icon_90x40-height: 40px;
 $icon_90x40-top: (-1 * $icon_100x30-height) + $icon_100x30-top;
-$icon_90x40-classes: visa mastercard amex;
+$icon_90x40-classes: visa mastercard amex wechat alipay_hk alipay;
```

Comment:
@tachyons Possible to add country prefixes for both WeChat and Alipay for clarity - `wechatpay_hk`, alipay_hk`, `alipay_cn`

Opens up the possibility of adding WeChatPay HK with a different logo and easier understand in the future.

Reply:
https://github.com/sephora-sea/sephora-style-guide/pull/87#discussion_r305693663

Reply:
@tachyons Replied in that thread!